usaspending_api/etl/management/commands/elasticsearch_indexer_for_spark.py

```diff
@@ -20,6 +20,10 @@ class Command(AbstractElasticsearchIndexer):
     node instance sizes did.
     """
 
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.valid_load_types = {"transaction", "award", "recipient", "location", "subaward"}
```

Comment:
```suggestion
        self.valid_load_types = {*self.valid_load_types, "location"}
```
What are your thoughts on something like this? My thought process is that it keeps the types defined in `AbstractElasticsearchIndexer` as the "defaults".